tezster.js

```diff
@@ -10,8 +10,9 @@ const imageTag = "tezsureinc/tezster:1.0.0";
 const containerName = "tezster";
 
 program
-  .version("0.1.9", "-v, --version")
-  .command("setup")
+  .version('0.2.0', '-v, --version')
```

Comment:
Do not commit  the version change on dev branch. You can change locally for your testing.